components/fxa-client/src/internal/state_persistence.rs

```diff
@@ -135,156 +132,10 @@ impl StateV2 {
     }
 }
 
-/// Migration from `StateV1` to `StateV2`.
-/// There was a lot of changing of structs and renaming of fields,
-/// but the key change is that we went from supporting multiple active
-/// refresh_tokens to only supporting a single one.
-///
```

Comment:
In place of the implementation here, could you please leave a modified version of this comment, explaining that there *was* a `StateV1` that looked different, and it was deliberately removed? (Otherwise future readers of this code may *infer* the existence of a `StateV1` and spend needless cycles trying to work out where the code for it lives).

Reply:
Ended up putting this up in the module docs at the top - seemed more appropriate there 😄